src/olympia/devhub/templates/devhub/verify_email.html

```diff
@@ -0,0 +1,52 @@
+{% extends "devhub/base.html" %}
+
+{% set title = _('Email Verification') %}
+
+{% block title %}{{ dev_page_title(title) }}{% endblock %}
+
+{% block content %}
+<h1>{{ title }}</h1>
+<div id="{{ state }}">
+{% if state == "email_verified" %}
+  {% trans %}
+  Your email address verified.
+  {% endtrans %}
+{% elif state == "email_suppressed" %}
+  {% trans %}
+  Please verify your email by clicking "Verify email" above.
+  {% endtrans %}
+{% elif state == "verification_expired" %}
+  {% trans %}
+  Could not verify email. The verification link has expired.
+  {% endtrans %}
+{% elif state == "verification_pending" %}
+  {% trans %}
+  Working... Please be patient.
+  {% endtrans %}
+  <div class="loader"></div>
+{% elif state == "verification_failed" %}
+  {% trans %}
+  Failed to send confirmation email. Please try again.
+  If you no longer have access to your email address,
+  please update your mozilla account email address.
```

Comment:
```suggestion
  If you no longer have access to your email address, please update your Mozilla account email address.
```

Reply:
We should link to the Mozilla account profile page in this scenario.

Reply:
👍